packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx

```diff
@@ -251,7 +254,17 @@ const getDocs: () => JSX.Element = () => {
       <Canvas mdxSource={ScreenSharingFromViewerExampleText}>
         <ScreenSharingFromViewerExample />
       </Canvas>
-
+      <Heading>PowerPoint Live (PPTLive) Experience</Heading>
+      <Description>
+        The shared PPTLive is the only element placed in the [Grid
+        Layout](./?path=/docs/ui-components-gridlayout--grid-layout) and all remote participants are placed in the
+        horizontal gallery in the lower section. To be able to view this shared PPTLive, the sharing participant should
```

Comment:
@dmceachernmsft is always on the horizontal gallery a correct statement? should we call it the overflow if it could appear in a vertical style gallery instead?